### PR TITLE
Animegan demo

### LINKED ISSUE #442: GAN demo
Try AnimeGAN2 or another network to generate an augmented stream:

https://github.com/TachibanaYoshino/AnimeGANv2

### PR DESCRIPTION
**Pull request by abramov-oleg**

### CODE REVIEW

File: samples/animegan/animegan_converter.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
+from typing import Any, List, Optional, Tuple
+
+import cv2
+import numpy as np
+
+from savant.base.converter import BaseAttributeModelOutputConverter
+from savant.base.model import AttributeModel
+
+
+class AnimeganConverter(BaseAttributeModelOutputConverter):
```
Comment (denisvmedyantsev) [inline samples/animegan/animegan_converter.py:10]: We can add to savant/converter 
```
class ModelRawOutputConverter(BaseAttributeModelOutputConverter):
    def __call__(...) -> List[Tuple[str, Any, Optional[float]]]:
        return [
            (model.output.attributes[i].name, output, 1.0)
            for i, output in enumerate(output_layers)
        ]
```
And use it for such tasks (GAN, SuperResolution), moving needed model output scaling to overlay pyfunc.
@bwsw ?

Comment (denisvmedyantsev) [reply samples/animegan/animegan_converter.py:10]: There is such converter in https://github.com/insight-platform/Savant/pull/536/commits/29449b452c0d30114f48c0e4a9ab86e0341df01b

File: samples/README.md [hunk 2]
```diff
@@ -102,6 +102,14 @@ Preview:
 
 ![](opencv_cuda_bg_remover_mog2/assets/opencv_cuda_bg_remover_mog2-800.webp)
 
+### AnimeGAN demo
+
+Sample Location: [animegan](./animegan)
+
+Preview:
```
Comment (denisvmedyantsev) [inline samples/README.md]: Please, remove "Preview:".

File: samples/animegan/README.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+Tested on platforms:
+
+- Nvidia Turing, Ampere.
+
+Demonstrated adapters:
+
+- Multi-stream source adapter;
+- Video/Metadata sink adapter.
+
+**Note**: Ubuntu 22.04 runtime configuration [guide](../../docs/runtime-configuration.md) helps to configure the runtime to run Savant pipelines.
```
Comment (denisvmedyantsev) [inline samples/animegan/README.md]: Let's stop copy-paste this Note. I don't think it's useful, and the link is wrong.

File: samples/animegan/README.md [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+./samples/animegan/run.sh
+```
+
+## Change input video
+
+Input video is expected to be located in the `Savant/data` directory. File name can be set through `INPUT_FILENAME` environment variable
+
+```bash
+INPUT_FILENAME=input.mp4 ./samples/animegan/run.sh
+```
```
Comment (denisvmedyantsev) [inline samples/animegan/README.md:53]: And where can I find the result?

**Review by denisvmedyantsev — APPROVED** [review]

File: samples/animegan/animegan_converter.py (hunk 1 above)
Comment (abramov-oleg) [reply samples/animegan/animegan_converter.py:10]: I'll leave it as is for this demo.

File: samples/animegan/README.md (hunk 4 above)
Comment (abramov-oleg) [reply samples/animegan/README.md:53]: Added a note about the result location.

File: samples/animegan/README.md (hunk 3 above)
Comment (abramov-oleg) [reply samples/animegan/README.md]: Removed.

File: samples/README.md (hunk 2 above)
Comment (abramov-oleg) [reply samples/README.md]: Removed

**Pull request merged**